DataAccessLayer/IDataManager.cs

```diff
@@ -13,6 +13,9 @@ public interface IDataManager
 
         void UpdateData(string id, object catalogueItem);
 
+        //Justin's Code
+        void DeleteData(string id);
```

Comment:
Good! Now, if you go to build the project, you will see errors. This is expected. It just means that this change to the interface has to be implemented everywhere that the interface is used. The contract must be followed. So, you should need to make a change in the TestDataManager class in the test project as well as the GoogleSheetsDataManager. For the google sheets one you can just add a function for delete there and have it do nothing.

We want the test to work though so make sure to try to add something that calls a remove on the list in the testdatamanager.

Then you can finally add a test for it. Remove the default item from the list and then verify the item is no longer there.